lib/helpers.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const globalsSet = require('./globals-set.js');
+let IMPORTS = {};
+
+/**
+ * Adds an import for the module to the list
+ *
+ * @param {string} mod - the name of the module to import from
+ * @param {string} original - the name of the original to get
+ */
+function updateImports(mod, original) {
+  if (mod in IMPORTS) {
+    if (original in IMPORTS[mod]) {
+      // if an import of original from module exists, noop
+      return;
+    }
+    // if an import from module exists without importing original,
+    // add original to the import
+    IMPORTS[mod].append(original);
+    return;
+  }
+
+  // if no import from module exists, add a new import
+  // of original, from module
+  IMPORTS[mod] = [original];
+}
+
+/**
+ * Adds all imports from the list to the top of the source file
+ *
+ * @param {SourceFile} sourceFile - the file to add the imports to
+ */
+function addAllImports(sourceFile) {
+  for (const module in IMPORTS) { // eslint-disable-line guard-for-in
```

Comment:
You can do `for (const [module, import] of Object.entries(IMPORTS)) { ... }`, and avoid the ESLint warning.